internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -613,6 +621,7 @@ func resourceCosmosDbAccountUpdate(d *pluginsdk.ResourceData, meta interface{})
 	enableAutomaticFailover := d.Get("enable_automatic_failover").(bool)
 	enableMultipleWriteLocations := d.Get("enable_multiple_write_locations").(bool)
 	enableAnalyticalStorage := d.Get("analytical_storage_enabled").(bool)
+	localAuthenticationDisabled := d.Get("local_authentication_disabled").(bool)
```

Comment:
This could be named like below to improve readability
```suggestion
	disableLocalAuthentication := d.Get("local_authentication_disabled").(bool)
```

Reply:
Thanks for the review. Variable name has been modified to improve readability.